content/docs/studio/user-guide/model-registry/assign-stage.md

```diff
@@ -0,0 +1,55 @@
+# Assign stages to models
+
+To manage model lifecycle, you can assign stages (such as `development`,
+`staging`, `production`, etc.) to specific model versions.
+
+To assign a stage to a model version, Iterative Studio uses [GTO] to create an
+[annotated Git tag][git tag] with the specified stage and version number. Refer
+the [GTO docs][gto] to see the exact format of the Git tag.
+
+You can write CI/CD actions that can actually deploy the models to the different
+deployment environments upon the creation of a new Git tag for stage assignment.
+For that, you can leverage any ML model deployment tool, such as MLEM.
+
+You can assign stages using the [GTO] CLI. To assign stages using Iterative
+Studio, watch this tutorial video or read on below:
+
+https://youtu.be/Vrp1O5lkWBo
```

Comment:
- [ ] Make video public (and change link to https://www.youtube.com/watch?v=Vrp1O5lkWBo ?)